### PR TITLE
CamembertForCausalLM

### PR DESCRIPTION
**Pull request by patil-suraj**
This PR adds `CamembertForCausalLM` by subclassing `RobertaForCausalLM`, so that it can be used with the `EncoderDecoderModel`.

@patrickvonplaten

### CODE REVIEW

**Comment by patil-suraj** [comment]
@patrickvonplaten  didn't add any tests since it subclasses `RobertaForCausalLM` which is already tested.

**Comment by patil-suraj** [comment]
hi @patrickvonplaten , could you take a look ? Thanks !

**Review by JetRunner — COMMENTED** [review]
I'm a little curious that why this works? Camembert and Roberta are trained by masking so technically it's not very good for casual language modeling, right?

**Comment by patil-suraj** [comment]
Yes, right. But this intended for the `EncoderDecoder` model class which can use pre-trained encoder for seq2seq tasks.
See PR #6411 and #6538 . Also Patrick shared this awesome paper which explores this idea and has shown some good results on seq2seq tasks.  https://arxiv.org/pdf/1907.12461.pdf

**Comment by JetRunner** [comment]
Thanks for the explanation! Yes it makes perfect sense to use these models for seq2seq but I still think maybe we should add a note somewhere in the document to explain that?

**Comment by patil-suraj** [comment]
Definitely! Will update the docs to explicitly state that this is intended for seq2seq and might not perform well on just causal modelling.  Will also link the paper in `EncoderDecoder` doc page, so people will know what to choose.

**Review by patrickvonplaten — APPROVED** [review]
Great, thanks @patil-suraj !
